e2e/terraform/gke/oebs-staging.tf

```diff
@@ -0,0 +1,29 @@
+
+provider "google" {
```

Comment:
The code is the place where we should strive to provide maximum comments that lets others understand the essentials. In short, can we have good comments?

Reply:
Sure. Added comments @AmitKumarDas